include/fmt/ranges.h

```diff
@@ -445,7 +445,7 @@ struct range_formatter<
   FMT_CONSTEXPR auto parse(ParseContext& ctx) -> decltype(ctx.begin()) {
     auto it = ctx.begin();
     auto end = ctx.end();
-    detail::maybe_set_debug_format(underlying_, true);
+    //detail::maybe_set_debug_format(underlying_, true);
```

Comment:
I end up with something like: '\'''A''\'''\'''s''\'''\'''s''\'''\'''e''\'''\'''r''\'''\'''t''\ etc. because of that unconditional debug format setting.
I use `{}` as format specifier.

Reply:
You'll need to set it conditionally for non-string kinds.